src/inc/quic_crypt.h

```diff
@@ -84,7 +84,7 @@ QuicKeyLength(
     case QUIC_AEAD_AES_256_GCM:
```

Comment:
QUIC_AEAD needs to be updated.

Reply:
Do they? This and QUIC_HASH seem very QUIC specific.

Reply:
Only QUIC_PACKET stuff is QUIC specific. Otherwise they are all fairly generic crypto abstractions.